core/package.json

```diff
@@ -11,23 +11,18 @@
   "author": "Nuxeo",
   "license": "Apache 2.0",
   "scripts": {
-    "test": "polymer test --npm --simpleOutput --expanded -l chrome,firefox",
-    "docs": "polymer analyze -i nuxeo-elements.js > analysis.json && polymer serve",
-    "coveralls": "cat coverage/lcov.info | coveralls"
+    "docs": "polymer analyze . > analysis.json && npm run serve",
```

Comment:
Is this running? We recently faced some issues with `polymer analyze` and we add the need to add `-i X` in order to workaround its automatic finding feature that seems to be buggy.

Reply:
Note the added `.`. Polymer analyser should read our package.json and use the "main" as input.

Reply:
😅
Should, indeed.